index.html

```diff
@@ -158,41 +158,51 @@ <h4>Profesional Art Printing Data More</h4>
     </div>
     <!------------------about me------------------------------------------------>
     <div class="aboutme-container">
-                <div class="aboutme-left">
-                <h4>About me</h4>
-                <p>Hey there! I can help you build a product, feature or website. Look through some of my work and
-                    experience! If you like
-                    what you see and have a project you need coded, don’t hesitate to contact me.</p>
-            </div>
-            <div class="aboutme-right">
-                <button class="gr">Get My Resume</button>
-            </div>
-
+              <div class="aboutme-row1">
+                    <div class="aboutme-left">
+                    <h4>About me</h4>
```

Comment:
- [ ] Kindly update the `font-family` of this heading to match the design. 😎️

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/17243325/214768526-4a702b42-20f9-43c0-9eda-6b084e45552b.png" width="300">|<img src="https://user-images.githubusercontent.com/17243325/214768527-a87e11cb-16ef-41cc-a847-ed3c95749313.png" width="300">|